include/drivers/flash/flash_rpc.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef FLASH_RPC_H_
+#define FLASH_RPC_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum flash_rpc_command {
+	RPC_COMMAND_FLASH_INIT = 0x01,
+	RPC_COMMAND_FLASH_READ = 0x02,
+	RPC_COMMAND_FLASH_WRITE = 0x03,
+	RPC_COMMAND_FLASH_ERASE = 0x04,
+};
```

Comment:
Question: is there some RPC namespace and commands we can collide with?

Reply:
I guess not. Getting familiar with NRF_RPC_GROUP_DEFINE

Reply:
I Could maybe do FLASH_RPC instead will change.